adlfs/multithread.py

```diff
@@ -327,7 +327,7 @@ def run(self, nthreads=None, monitor=True):
         for (rfile, lfile), dic in self.progress.items():
             unique = dic['uuid']
             if len(dic['waiting']) > 1:
-                parts = [self.temp_upload_path+unique+"_%i" % i for i
+                parts = [self.temp_upload_path+unique+"/part_%i" % i for i
```

Comment:
Correct, the first file will result in the folder being created. We have no hard requirement on the underlying directory existing prior to file creation.

---
In reply to: [78467170](https://github.com/Azure/azure-data-lake-store-python/pull/63#discussion_r78467170) [](ancestors = 78467170)